terraform/environments/corporate-staff-rostering/locals_security_groups.tf

```diff
@@ -546,6 +558,470 @@ locals {
+          from_port   = 0
+          to_port     = 0
+          protocol    = -1
+          cidr_blocks = local.security_group_cidrs.https
+        } */
+        http = {
+          description = "Allow http ingress"
+          from_port   = 80
+          to_port     = 80
+          protocol    = "TCP"
+          cidr_blocks = local.security_group_cidrs.https
+        }
+        https = {
+          description = "Allow https ingress"
+          from_port   = 443
+          to_port     = 443
+          protocol    = "TCP"
+          cidr_blocks = local.security_group_cidrs.https
+        }
+        http7770_7771_lb = {
+          description = "Allow http 7770-7771 ingress"
+          from_port   = 7770
+          to_port     = 7771
+          protocol    = "TCP"
+          cidr_blocks = local.security_group_cidrs.http7xxx
+        }
+        http7780_7781_lb = {
+          description = "Allow http 7780-7781 ingress"
+          from_port   = 7780
+          to_port     = 7781
+          protocol    = "TCP"
+          cidr_blocks = local.security_group_cidrs.http7xxx
+        }
+      }
+      egress = {
+        all = {
+          description = "Allow all traffic outbound"
+          from_port   = 0
+          to_port     = 0
+          protocol    = "-1"
+          cidr_blocks = ["0.0.0.0/0"]
+        }
+      }
+    }
+    web = {
+      description = "New security group for web-servers"
+      ingress = {
+        all-from-self = {
+          description = "Allow all ingress to self"
+          from_port   = 0
+          to_port     = 0
+          protocol    = -1
+          self        = true
+        }
+        # IMPORTANT: check if an 'allow all from load-balancer' rule is required
+        http_web = {
+          description     = "80: http allow ingress"
+          from_port       = 80
+          to_port         = 80
+          protocol        = "TCP"
+          cidr_blocks     = ["10.0.0.0/8"]
+          security_groups = ["load-balancer"]
+          # NOTE: will need to be changed to point to client access possibly
+        }
+        rpc_tcp_web = {
+          description     = "135: TCP MS-RPC allow ingress from app and db servers"
+          from_port       = 135
+          to_port         = 135
+          protocol        = "TCP"
+          security_groups = ["app", "db"]
+          # NOTE: csr_clientaccess will need to be added here to cidr_blocks
+        }
+        rpc_tcp_web = {
+          description     = "135: UDP MS-RPC allow ingress from app and db servers"
+          from_port       = 135
+          to_port         = 135
+          protocol        = "UDP"
+          security_groups = ["app", "db"]
+          # NOTE: csr_clientaccess will need to be added here to cidr_blocks
+        }
+        https_web = {
+          description     = "443: https ingress"
+          from_port       = 443
+          to_port         = 443
+          protocol        = "TCP"
+          cidr_blocks     = ["10.0.0.0/8"]
```

Comment:
Use local.security_group_cidrs.https for consistency

Reply:
changes to enduserclient